assets/js/base/components/checkout/form-step/style.scss

```
@@ -0,0 +1,69 @@
+$inner-circle-size: 24px;
+$circle-border-size: 8px;
+$outer-circle-size: $inner-circle-size + (2 * $circle-border-size);
+
+// @todo: replace those colors with the correct values once we settle on a design system palette.
+$gray-20: #c3c4c7;
+$gray-20: #a7aaad;
+$gray-60: #50575e;
+$gray-80: #2c3338;
+
+
+.wc-components-checkout-step {
+	position: relative;
+	border-left: 1px solid $gray-20;
+	padding-left: $gap-large;
+	padding-bottom: $gap-larger;
+
+	&.is-last-step {
+		border-left: none;
+	}
+}
+
+.wc-components-checkout-step__heading {
+	display: flex;
+	justify-content: space-between;
+	align-content: center;
+}
+
+// @todo: remove the parent class once we dial the specification down.
+.wc-components-checkout-step__heading .wc-components-checkout-step__title {
+	@include font-size(16, 24px);
+	color: $gray-80;
+	margin-bottom: $gap-smaller;
+	font-weight: 400;
+}
+
+// @todo: remove the parent class once we dial the specification down.
+.wc-components-checkout-step .wc-components-checkout-step__secondary-action {
+	@include font-size(12, 22px);
```

Should this have the same `line-height` as the title so they are aligned?
```suggestion
	@include font-size(12, 24px);
```

---

line-height doesn’t matter much here since they’re aligned with flex but yeah.